OpenRA.Mods.Common/Traits/Production.cs

```diff
@@ -118,17 +118,11 @@ public virtual bool Produce(Actor self, ActorInfo producee, string factionVarian
 			if (Reservable.IsReserved(self))
 				return false;
 
-			if (!occupiesSpace)
```

Comment:
See the other change in this patch, which prevents modders from defining exits on actors that don't occupy space.